src/styles/main.scss

```diff
@@ -0,0 +1,59 @@
+.container {
+  width: 80vmin;
+  height: 80vmin;
+  border: 120px dotted black;
+  border-radius: 50%;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 80px auto;
+}
+
+.circle {
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  background-color: orange;
+  position: absolute;
+  left: 335px;
+  z-index: 1;
+}
+
+.hours {
+  width: 280px;
+  height: 30px;
+  background-color: blue;
+  position: absolute;
+  left: 350px;
+  transform-origin: left;
+  animation: rotate-sec 36000s infinite steps(12);
+}
+
+.seconds {
+  width: 20px;
+  height: 380px;
+  background-color: green;
+  position: absolute;
+  left: 350px;
+  bottom: 360px;
+  transform-origin: bottom;
+  animation: rotate-sec 60s infinite steps(60);
+}
+
+.minutes {
+  width: 20px;
+  height: 380px;
+  background-color: red;
+  position: absolute;
+  left: 350px;
```

Comment:
same problem for all arrows
center all arrows as circle in comment
![Screenshot from 2023-06-02 12-22-18](https://github.com/mate-academy/layout_stop-watch/assets/33029864/052a337b-dc54-48f5-ba9c-068d22f2e794)

Reply:
hours, seconds and minutes have almost same styles.
consider to make mixin for them which will take parameteres which styles are different
and pay attention on task description
enough minute and second arrow and it shouldn't be real rotation for testing
![Screenshot from 2023-06-02 12-17-08](https://github.com/mate-academy/layout_stop-watch/assets/33029864/b3051e9d-50f7-4df1-a521-892f0634270c)